addons/pos_sale/views/product_views.xml

```diff
@@ -1,15 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-
+    //TODO: remove this view in master
```

Comment:
Could you use XML syntax for this comment? I'm not sure what will happen if you leave this as is :sweat_smile: